template/service/running.sls

```diff
@@ -0,0 +1,17 @@
+# -*- coding: utf-8 -*-
+# vim: ft=sls
+
+{#- Get the `topdir` from `tpldir` #}
+{%- set topdir = tpldir.split('/')[0] %}
+{%- set sls_config_file = topdir ~ '.config.file' %}
+{%- from topdir ~ "/map.jinja" import template with context %}
+
+include:
+  - {{ sls_config_file }}
+
+template-service:
```

Comment:
More descriptive state name is `template service running enabled`.

Reply:
OK.